login/routers/SignUp.js

```diff
@@ -2,10 +2,11 @@ const express = require('express');
 
 const router = express.Router();
 const RegistrationRequest = require('../db/db_objects/RegestrationRequest.js');
-const signUpValidation = require('../core/validations/SignUpValidation.js');
+const UserExistCheck = require('./controllers/UserExistCheck.js');
+
 
 router.post('/', (request, response) => {
-  const registrationRequest = new RegistrationRequest(request.body.login, request.body.mail, request.body.password);
-  signUpValidation.SignUpValidation(registrationRequest, response);
+  const registrationRequest = new RegistrationRequest(request.body.login, request.body.email, request.body.password);
+  UserExistCheck.UserExist(registrationRequest, response);
```

Comment:
Does the name UserExistCheck really represent what the class is about? I had to look for it to understand what this class was about and what I found was a method that created a user account or threw an error if a user like that already existed, a name like UserService would better describe a class that has that sort of functionality.